internal/services/securitycenter/security_center_subscription_pricing_resource.go

```diff
@@ -147,6 +180,9 @@ func resourceSecurityCenterSubscriptionPricingRead(d *pluginsdk.ResourceData, me
 		if properties := resp.Model.Properties; properties != nil {
 			d.Set("tier", properties.PricingTier)
 			d.Set("subplan", properties.SubPlan)
+			if properties.Extensions != nil {
+				d.Set("extensions", *(properties.Extensions))
```

Comment:
when setting property blocks in the read function we normally create a function to do this - in this case, something like `flattenSecurityCenterSubscriptionPricingExtensions`. [Here](https://github.com/hashicorp/terraform-provider-azurerm/blob/0cd8f8c6c3ab8d81d640c02d4618c1033224826e/internal/services/datadog/datadog_monitor_tag_rule_resource.go#L299) is an example of a flatten function from another resource.

Reply:
Done